content/open-source/frameworks/what-is-the-oci.md

```diff
@@ -0,0 +1,111 @@
+
+## Runtime Specification  
+
+A container runtime is the software used to run and manage containers; essentially they create and run containers using specified images. The goal of the OCI Runtime Specification is to ensure consistency between different runtime environments and define common actions to manage a container’s lifecycle. An OCI-compliant image should run reliably on any OCI-compliant runtime.  More information about the Runtime Specification can be found in the [OCI documentation](https://github.com/opencontainers/runtime-spec/blob/main/spec.md). 
+
+In addition to overseeing this specification, OCI develops the runtime `runc`, a command line client for creating, configuring, and managing containers. Originally developed by Docker, `runc` was donated to OCI as the reference for the specification and serves as the foundation for a variety of popular container tools such as containerd and Podman. 
+
+## Distribution Specification
+
+The OCI Distribution Specification aims to standardize the way container registries and runtime tools push and pull container images and other content types. It is based on the specification for the Docker Registry HTTP API V2 protocol apdx-1, and has been adopted by major registries such as Amazon Elastic Container Registry, Google Container Registry, Azure Container Registry, and Github Container Registry. Any registry that is OCI-compliant supports interactions outlined by this specification, such as pushing, pulling, or storing images.  More information about the Distribution Specification can be found in the OCI documentation [here](https://github.com/opencontainers/distribution-spec/blob/main/spec.md). 
+
+## How to know if an image is OCI compliant
+
+Currently, most images encountered in the wild are either OCI or Docker images. You can determine whether an image is OCI compliant by inspecting the `mediatype` value located in the image’s manifest. If “oci” is included in the string set as the value of the `mediatype`, then it is an OCI image:
+
+```json
+ "mediaType": "application/vnd.oci.image.manifest.v1+json",
+```
+
+Otherwise, the mediatype string will likely include “docker” as follows: 
+
+```json 
+“mediaType”: “application/vnd.docker.distribution.manifest.v2+json”
+```
+
+There are a few interesting nuances about OCI images that are worth pointing out. First, because Docker donated its image specifications to OCI, Docker and OCI image specifications are the same in substance. In fact, most images on Docker are _Docker_ images and not OCI images, which you can confirm by inspecting the image manifests. This is largely due to the fact that Docker’s tools for publishing and building images create _Docker_ images – not _OCI_ images – by default, a convention set by historical practice. If you want to build and publish OCI images, you must use tools that support OCI, such as [apko](https://edu.chainguard.dev/open-source/apko/overview/), an open source OCI image builder.  
```

Comment:
If so, can you indeed clarify that you _cannot_ build OCI images with Docker?

Also make sure those are em dashes please 😄

Reply:
Josh seemed to think that you couldn't build OCI images on Docker, but it was complicated. It might be best to not say anything on that point at this moment.